src/connect.rs

```diff
@@ -0,0 +1,57 @@
+use bevy::prelude::*;
+use bevy_egui::{egui, EguiContext};
+
+use crate::{GameState, MySystemLabel};
+
+const INITIAL_URL: &str = "http://localhost:8080";
```

Comment:
Maybe use different port, because this one is used by many projects.

Reply:
this is just the default port of the whist-server application in dev mode